common.gradle

```
@@ -0,0 +1,106 @@
+apply plugin: 'application'
+apply plugin: 'java-library'
+
+sourceCompatibility = 8
+targetCompatibility = 8
+
+def rapidwrightDir = System.getenv("RAPIDWRIGHT_PATH") ? System.getenv("RAPIDWRIGHT_PATH") : file('.').toString()
```

```suggestion
def rapidwrightDir = System.getenv("RAPIDWRIGHT_PATH") ?: file('.').toString()
```
Just learned that Groovy has a more liberal definition of `True` than Java.  This led me to discover the ["Elvis operator"](https://groovy-lang.org/operators.html#_elvis_operator).